### ISSUE TITLE
Reserve protocol used usernames

### ISSUE THREAD

**Issue by pellaeon** [pellaeon/registration#113]
- https://github.com/shouldbee/reserved-usernames
- https://zimbatm.github.io/hostnames-and-usernames-to-reserve/
- https://github.com/shouldbee/reserved-usernames

**Comment by buffcode** [comment]
Do ownCloud or Nextcloud already have some checks in place in order to deny creation of blocked user names? I think this should be handled within the core, otherwise a blocked username could still be created using the backend.

Alternatively these lists could be used to prefill a list of blocked usernames in the backend, which can then be administered.

**Comment by pellaeon** [comment]
Good point, I will look into that, but also in some cases it would be reasonable to allow admins to create such usernames but refuse self-registered users to choose these usernames.

**Comment by cdamken** [comment]
Probably will also good to deny some characters, like space
